modules/navier_stokes/src/userobjects/INSADObjectTracker.C

```diff
@@ -77,3 +77,10 @@ addAmbientConvectionParams(InputParameters & params)
                         "The heat transfer coefficient from from ambient surroundings");
   params.addParam<Real>("ambient_temperature", "The ambient temperature");
 }
+
+template <>
+bool
+INSADObjectTracker::notEqual(const MooseEnum & val1, const MooseEnum & val2)
```

Comment:
Is it necessary to create this function? The diff indicates that we only need to change one place, which we can just directly use `compareCurrent`.

Reply:
If I don't have this function then I have to do a whole specialization for `INSADObjectTracker::set` for `MooseEnum`, which is fine if that's what you prefer. Just more duplicate code (although only 10 lines)